nservicebus/hosting/service-fabric-hosting/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Service Fabric Hosting
+related:
+ - nservicebus/service-fabric
+ - samples/azure/azure-service-fabric-routing
+reviewed: 2017-03-29
+---
+
+Hosting with Reliable Services general info.
+
+### Stateless services
+
+### Stateful service
+
+### Guest Executable
```

Comment:
Could someone also have the question:

Can I host NSB in an actor system and using NSB in the actors?

I think it is a valid (but potentially dangerous) thought to have and we should address it